astropy/tests/pytest_doctestplus.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern import six
+
+import doctest
+import fnmatch
+import imp
+import os
+import re
+import sys
+
+import pytest
+
+from .output_checker import AstropyOutputChecker, FIX
+
+try:
+    import importlib.machinery as importlib_machinery
+except ImportError:  # Python 2.7
```

Comment:
@mohanagr , Travis CI indicates failures for Python 2.7. I suspect this is the cause here. So this PR needs to be patched to work in Python 2. Also, fix PEP8 test failure.